src/services/api.js

```diff
@@ -618,9 +618,14 @@ export default {
       return multipartRequest("/admin/uploads", formData)
     },
 
-    delete(file) {
-      const path = `/admin/uploads/delete`
-      return medusaRequest("POST", path, { file })
+    delete(fileKey) {
```

Comment:
@pKorsholm can I remove that and use your update from medusa react instead? we will require to upgrade the dependencies before delivering that in the release

Reply:
yeah you should be able to use the delete method

Reply:
Same for the download url?